src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -67,7 +67,7 @@
 def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlation_id=None, https_proxy="", http_proxy="", no_proxy="", proxy_cert="", location=None,
```

Comment:
will this check get affected because of the least priviledge scenario?
https://github.com/Azure/azure-cli-extensions/blob/aca06ac34f5e94cc95e58c0fc86939636eb6c6eb/src/connectedk8s/azext_connectedk8s/custom.py#L147

Reply:
Actually we had added "create" access on clusterrolebindings previously, so this check was passing. But now that access is also not required. So removed it from role requirements and made this check conditional